tests/xdc/advanced_visibility_test.go

```diff
@@ -169,6 +192,26 @@ func (s *esCrossDCTestSuite) TestSearchAttributes() {
 
 	// Wait for namespace cache to pick the change
 	time.Sleep(cacheRefreshInterval)
+	if !s.isElasticsearchEnabled() {
+		// When Elasticsearch is enabled, the search attribute aliases are not used.
+		_, err = client1.UpdateNamespace(tests.NewContext(), &workflowservice.UpdateNamespaceRequest{
+			Namespace: namespace,
+			Config: &namespacepb.NamespaceConfig{
+				CustomSearchAttributeAliases: map[string]string{
+					"Bool01":     "CustomBoolField",
+					"Datetime01": "CustomDatetimeField",
+					"Double01":   "CustomDoubleField",
+					"Int01":      "CustomIntField",
+					"Keyword01":  "CustomKeywordField",
+					"Text01":     "CustomTextField",
+				},
+			},
+		})
+		s.NoError(err)
+		// Wait for namespace cache to pick the UpdateNamespace changes.
+		time.Sleep(cacheRefreshInterval)
+	}
+
```

Comment:
Why not move this to `SetupSuite` where it adds the search attributes to ES?

Reply:
Apparently it tests namespace creation in the test. Probably it should be moved out.  One day.